cedar-policy-core/src/ast/entity.rs

```diff
@@ -346,6 +355,20 @@ impl Entity {
         })
     }
 
+    /// Create a new `Entity` with this UID, ancestors, and an empty set of attributes
+    /// Since it lacks any attributes, this method returns `Self` instead of `Result<Self>`
```

Comment:
```suggestion
    /// Create a new `Entity` with this UID, ancestors, and an empty set of attributes
    ///
    /// Since there are no attributes, this method does not error, and returns `Self` instead of `Result<Self>`
```